invenio_opendata/base/templates/helpers/text/glossary.json

```diff
@@ -1,405 +0,0 @@
-[
```

Comment:
Nothing on your side, me or @pamfilos shall look at fixing this while merging your PR.